pal/system-info/minigame/system-info.ts

```diff
@@ -197,12 +161,6 @@ class System {
     public onClose (cb: () => void) {
         this._eventTarget.on(AppEvent.CLOSE, cb);
     }
-    public onViewResize (cb: () => void) {
-        this._eventTarget.on(AppEvent.RESIZE, cb);
-    }
-    public onOrientationChange (cb: () => void) {
-        this._eventTarget.on(AppEvent.ORIENTATION_CHANGE, cb);
-    }
```

Comment:
SystemInfo should also extend from `EventTarget` too, please remove all these event handling APIs, it's not following the global coding  style

Reply:
updated